test/powershell/Host/Logging.Tests.ps1

```diff
@@ -317,26 +340,19 @@ Path:.*
         }
     }
 
-    It 'Verifies basic logging with no customizations' -Skip:(!$IsSupportedEnvironment) {
+    It 'Verifies basic logging with no customizations' -Skip:(!$IsMacOS) {
         try {
+            $timeString = [DateTime]::Now.ToString('yyyy-MM-dd HH:mm:ss')
             $configFile = WriteLogSettings -LogId $logId
+            copy-item $configFile /tmp/pwshtest.config.json
             $testPid = & $powershell -NoProfile -SettingsFile $configFile -Command '$PID'
-
-            Export-PSOsLog -After $after -LogPid $testPid -TimeoutInMilliseconds 30000 -IntervalInMilliseconds 3000 -MinimumCount 3 |
```

Comment:
`Export-PSSysLog` is no longer used, so maybe we should remove it from the `PSSysLog` module?